### ISSUE TITLE
Pen transparency bugs with thin lines rendering

### ISSUE THREAD

**Issue by AntoineTelgruc2** [LLK/scratch-render#993]
### Expected Behavior

I was making a color table for some reason, and I found a way to make the rendering fast drawing a lot of vertical color lines and drawing above horizontal lines with different pen transparency

### Actual Behavior

Time to time, while rendering the table, the transparent lines are making some wierd glitched bue and white "chunks"

### Steps to Reproduce

I shared a project featuring the bug you can see at https://scratch.mit.edu/projects/722213316 , and Chrome_Cat (a scratcher who has a lot of experience with pen) said me this in my comments:

"I can see the flickering. It occurs only if the pen line has transparency, is equal or longer than 379 pixels and pen size is less than 5. If you set the transparency to 0, the flickering increases. I expect it is an issue with how the gpu renders long thin pen lines. Potentially it skips pixels for some reason."

and he said me to submit this bug on github, so that's why I created an account to post this

### System Details

The GPU has potentially a role in that, because I have the bug on on my laptop (a 2020 huawei matebook with google chrome), but it doesn't seems to happen on my Xiaomi note 9 (with google chrome too)

**Screenshots**

[video (23).webm](https://user-images.githubusercontent.com/111634603/185739323-6ee9bd8f-5207-42bf-a483-2b246ca96838.webm)

**Comment by adroitwhiz** [comment]
The fact that this doesn't happen consistently (rendering the exact same line in the exact same place with the same thickness and color will sometimes work and sometimes glitch) suggests that this is a bug with your graphics card or graphics driver. Unfortunately, there's not much that can be done about that on Scratch's end.

**Comment by cwillisf** [comment]
I agree with @adroitwhiz that it's probably a bug in the GPU or its driver. In some cases, we're able to work around this kind of bug by adjusting the way we draw things. I can't be sure that we'll be able to in every case, but I appreciate the report!
